preprocess:
    reference_cloud_uri: /home/navsea-jetson/catkin_ws/src/navsea/resource/mean_cloud.pcd  
    icp_threshold: 0.5
    icp_tf_init: [[1,  0,  0,0],[0,  1,0,0],[0, 0, 1 ,0], [0,0,0,1]]
    denoise_neighbor: 50
    denoise_std: 2.0
    downsample_rate: 0.1
    zbound: 0.5

fod_detection:
    cloud_info_uri: /home/navsea-jetson/catkin_ws/src/navsea/resource/mean_cloud_info.p    
    metric: mahalanobis
    FOD_minsize: 5

smoothing:
    method: knn
    k: 50 #100
    iteration: 1 #4

clustering:
    minimum_fod_point_count: 2
    cluster_voxel_size: 0.05